packages/server/lib/plugins/child/ts_node.js

```diff
@@ -3,18 +3,41 @@ const debugLib = require('debug')
 const path = require('path')
 const tsnode = require('ts-node')
 const resolve = require('../../util/resolve')
+const semver = require('semver')
 
 const debug = debugLib('cypress:server:ts-node')
 
 const getTsNodeOptions = (tsPath, registeredFile) => {
+  const compiler = process.env.TS_NODE_COMPILER || tsPath
+
+  const version = require(compiler).version
```

Comment:
What version of typescript does `typescript-cached-transpile` use? do you know?

Reply:
Maybe I should simply ignore TS_NODE_COMPILER and do it using `tsPath` all the time. 

I want to make sure I use the version of typescript installed in the targetted repo, not the one where cypress is installed. Otherwise, if we use global mode, it will crash opening an `npm init vue` project